code/modules/mob/living/carbon/examine.dm

```diff
@@ -195,7 +195,7 @@
 	if(reagents.has_reagent(/datum/reagent/teslium, needs_metabolizing = TRUE))
 		. += span_smallnoticeital("[t_He] [t_is] emitting a gentle blue glow!") // this should be signalized
 
-	if(just_sleeping)
+	if(just_sleeping || appears_dead)
 		. += span_notice("[t_He] [t_is]n't responding to anything around [t_him] and seem[p_s()] to be asleep.")
```

Comment:
Extremely ambiguous message with no hint telling you to check the pulse, or how to do so. We discussed this several times before you even made this PR and you just ignored that.